ironfish-rust-nodejs/src/structs/note.rs

```diff
@@ -28,13 +28,14 @@ pub const GENERATOR_LENGTH: u32 = GENERATOR_SIZE as u32;
 pub const AMOUNT_VALUE_LENGTH: u32 = AMOUNT_VALUE_SIZE as u32;
 
 #[napi]
-pub const DECRYPTED_NOTE_LENGTH: u32 = 136;
+pub const DECRYPTED_NOTE_LENGTH: u32 = 168;
```

Comment:
Are these constants in our codebase? Can we add them here instead of manually updating this bytes value?